File: trunk/unityClientSample/Assets/src/protocol/SkillInfo.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
	[ProtoContract]
	public class SkillInfo
	{
        /// <summary>
        ///  技能id
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public string id;
        /// <summary>
        ///  受影响的卡牌id
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public List<string> target;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/domain/SkillInfo.java
/**
 * 卡牌
 */
package org.ngame.protocol.domain;



import java.util.List;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

import lombok.Data;

@Data
public class SkillInfo
{
  @Protobuf(description = "技能id")
  private String uniqueId;
  @Protobuf(description = "技能id")
  private String id;
  @Protobuf(description = "受影响的卡牌id")
  private List<String> target;

}
